packages/adminrouter/extra/src/includes/server/master.conf

```diff
@@ -439,9 +439,9 @@ location /acs/api/v1 {
 
 # Group: DC/OS UI Update Service
 # Description: DC/OS UI Update Service API
-location ~ ^/dcos-ui-update-service/api(?<url>.*)$ {
+location ~ ^/dcos-ui-update-service/api(?<api_path>.*)$ {
```

Comment:
We need `/api` to be `/api/` here or update the regex in the EE PR to expect a `/` at the beginning of the `api_path` parameter.